components/event-sources/reconciler/httpsource/controller_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	. "github.com/kyma-project/kyma/components/event-sources/reconciler/testing"
 
 	// Link fake clients accessed by reconciler.Base
+	fakeauthclient "istio.io/client-go/pkg/clientset/versioned/fake"
```

Comment:
Please separate injected clients from non-injected clients, otherwise the comment doesn't make sense anymore.

Reply:
This line is removed in the refactoring.